model/message.go

```diff
@@ -33,6 +46,16 @@ type Message struct {
 	//
 	// example: 2
 	Priority int `form:"priority" query:"priority" json:"priority"`
+	// The extra data sent along the message.
+	//
+	// The extra fields are stored in a key-value scheme. Only accepted in CreateMessage requests with application/json content-type.
+	//
+	// The keys should be in the following format: <top-namespace>::[<sub-namespace>::]<action>
```

Comment:
In the swagger ui only:
```
The keys should be in the following format: <top-namespace>::[<sub-namespace>::]
```
is displayed, I guess it interprets `<action>` as html.

Reply:
I have escaped them and they looked fine now